### PR TITLE
Don't reverse MatchParen highlighting

### PR DESCRIPTION
**Pull request by tannerwelsh**
Reversing the highlighting makes cursor movement hard to follow.

**Current**
Staring with cursor here...

Screen Shot 2020-11-24 at 5 54 53 PM

And then press `l`, where is the cursor now?

Screen Shot 2020-11-24 at 5 54 56 PM

B/c the light bg is used for the matching paren, the cursor appears to have jumped to the matching paren instead of to the (actual) cursor location over the `(`.

And w/ light background:

Screen Shot 2020-11-24 at 5 55 21 PM
Screen Shot 2020-11-24 at 5 55 24 PM

**Changes**
After this change, the cursor location is more obvious:

Screen Shot 2020-11-24 at 5 55 47 PM
Screen Shot 2020-11-24 at 5 56 00 PM

### CODE REVIEW

**Comment by tannerwelsh** [comment]
Hi @jsit - really like your theme, but I noticed my eye getting distracted by the handling of `MatchParen`. Changed it locally, thought I'd share the change here in case you/others have found the same issue.

**Comment by jsit** [comment]
Thanks!
